macstadium-shared-2/main.tf

```diff
@@ -0,0 +1,195 @@
+variable "macstadium_vanilla_image" { default = "travis-ci-ubuntu14.04-internal-vanilla-1481140635" }
+variable "jupiter_brain_prod_version" { default = "v0.2.0-58-gce0b45a" }
+variable "jupiter_brain_custom-1_version" { default = "v0.2.0-58-gce0b45a" }
+variable "jupiter_brain_custom-2_version" { default = "v0.2.0-58-gce0b45a" }
+variable "jupiter_brain_custom-3_version" { default = "v0.2.0-58-gce0b45a" }
+variable "jupiter_brain_staging_version" { default = "v0.2.0-58-gce0b45a" }
+variable "travis_worker_prod_version" { default = "v2.6.2" }
+variable "travis_worker_custom-1_version" { default = "v2.6.2" }
+variable "travis_worker_custom-2_version" { default = "v2.6.2" }
+variable "travis_worker_custom-3_version" { default = "v2.6.2" }
+variable "travis_worker_staging_version" { default = "v2.6.2" }
+variable "vsphere_janitor_version" { default = "9bde41b" }
+variable "collectd_vsphere_version" { default = "b648672" }
+variable "ssh_user" {
+  description = "your username on the wjb instances"
+}
+variable "threatstack_key" {}
+variable "librato_email" {}
+variable "librato_token" {}
+variable "collectd_vsphere_collectd_network_user" {}
+variable "collectd_vsphere_collectd_network_token" {}
+variable "fw_ip" {}
+variable "fw_snmp_community" {}
+variable "vsphere_user" {}
+variable "vsphere_password" {}
+variable "vsphere_server" {}
+
+provider "aws" {}
+provider "vsphere" {
+  user           = "${var.vsphere_user}"
+  password       = "${var.vsphere_password}"
+  vsphere_server = "${var.vsphere_server}"
+  allow_unverified_ssl = true
+}
+
+module "macstadium_infrastructure" {
+  source = "../modules/macstadium_infrastructure"
+  index = "${var.index}"
+  vanilla_image = "${var.macstadium_vanilla_image}"
+  datacenter = "TravisCI - Cluster_2"
+  cluster = "MacPro_Cluster"
+  datastore = "EMC-VMAX-1"
+  internal_network_label = "dvPortGroup-Internal"
+  management_network_label = "dvPortGroup-Mgmt"
+  jobs_network_label = "dvPortGroup-Jobs"
+  wjb_num = 2
+  ssh_user = "${var.ssh_user}"
+  threatstack_key = "${var.threatstack_key}"
+}
+
+module "jupiter_brain_prod_com" {
+  source = "../modules/jupiter_brain_bluegreen"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_ip_address = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.jupiter_brain_prod_version}"
+  config_path = "${path.module}/config/jupiter-brain-prod-com-env"
+  env = "com-prod"
+  index = "${var.index}"
+  port_suffix = 3
+}
+
+module "jupiter_brain_staging_com" {
+  source = "../modules/jupiter_brain_bluegreen"
+  host_id = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_ip_address = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user = "${var.ssh_user}"
+  version = "${var.jupiter_brain_prod_version}"
+  config_path = "${path.module}/config/jupiter-brain-staging-com-env"
+  env = "com-staging"
+  index = "${var.index}"
+  port_suffix = 4
+}
+
+module "jupiter_brain_custom_1" {
```

Comment:
I think the custom workers and jupiter brains should be removed for now, since they are currently only in one pod.